paper/paper.bib

```diff
@@ -39,12 +36,9 @@ @inproceedings{wave_geometry
 
 @misc{eigenweb,
 	Author = {Ga\"{e}l Guennebaud and Beno\^{i}t Jacob and others},
-	Howpublished = {[online] \url{http://eigen.tuxfamily.org}},
 	Title = {Eigen v3},
-	Url = {http://eigen.tuxfamily.org/index.php?title=Main_Page},
-	Urldate = {2018-12-02},
+	Url = {http://eigen.tuxfamily.org},
 	Year = {2010},
```

Comment:
```suggestion
	Year = {2010},}
```
`eigenweb` is now missing its closing parenthesis `}`.

Reply:
fixed